addons/medical/stringtable.xml

```diff
@@ -4108,14 +4108,22 @@
             <English>Whether or not the object will be a medical vehicle.</English>
             <Polish>Czy pojazdy ma być pojazdem medycznym?</Polish>
             <Italian>L'oggetto in questione sarà un veicolo medico o meno.</Italian>
-            <German>Ist das Objekt ein Sanitätsfahrzeug?</German>
+            <German>Legt fest, ob das Objekt ein Sanitätsfahrzeug ist.</German>
             <Spanish>Es un vehículo médico?</Spanish>
         </Key>
         <Key ID="STR_ACE_Medical_delayUnconCaptive">
-            <English>Delay Medical Unconscious Captivity</English>
-            <German>Verzögert Gefangenschaft bei medizinischer Bewustlosigkeit</German>
+            <English>Delay cease fire of AI during medical unconsciousness</English>
+            <German>Verzögert Ende des KI-Beschusses bei medizinischer Bewustlosigkeit</German>
             <Spanish>Demora antes de volverse neutral al caer inconsciente</Spanish>
             <Polish>Opóźnij status captive u nieprzytomnych osób</Polish>
         </Key>
+        <Key ID="STR_ACE_Medical_MedicalSettings_delayUnconCaptive_Description">
+            <English>Delay cease fire of AI while player is unconscious for medical reasons.</English>
+            <German>Verzögert das Ende des KI-Beschusses auf einen Spieler, wenn dieser aus medizinischen Gründen bewustlos wird.</German>
+        </Key>
+        <Key ID="STR_ACE_Medical_MedicalSettings_delayUnconCaptive_DisplayName">
+            <English>Delay cease fire of AI during medical unconsciousness</English>
```

Comment:
Changed display name according to your suggestion. Is the description length ok or does it need shortening too?